qutebrowser/browser/urlmarks.py

```diff
@@ -272,12 +272,17 @@ def _parse_line(self, line):
         elif len(parts) == 1:
             self.marks[parts[0]] = ''
 
-    def add(self, url, title):
+    def add(self, url, title, toggle=False):
         """Add a new bookmark.
 
+        Return True if the bookmark was added, and False if it was
```

Comment:
For consistency with other docstrings, please use "Return:" as header (like with "Args:") and then add the description indented below that.